Gemfile

```diff
@@ -6,11 +6,13 @@ git_source(:github) do |repo_name|
 end
 
 gem 'activeadmin'
+gem 'activeadmin_addons'
 gem 'devise', '~> 4.4.1'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.1.4'
 gem 'rubocop', '~> 0.52.1', require: false
 # Use postgresql as the database for Active Record
+gem 'paperclip', '~> 6.0.0'
```

Comment:
Paperclip is deprecated and will not be supported anymore: https://robots.thoughtbot.com/closing-the-trombone

It's better to switch to carrierwave

Reply:
@isabloCh, 
![image](https://user-images.githubusercontent.com/11134371/40498138-d5ff7b16-5f86-11e8-8830-07eb0b67bbf3.png)